tests/test_helper.py

```diff
@@ -58,3 +59,71 @@ def test_get_time_batches():
         datetime(2023, 11, 6, 15, 40, 17, 630499),
     )
     assert result[9] == (datetime(2023, 11, 6, 15, 40, 17, 630499), b)
+
+def test_array_dataframe():
+    submissions = [
+        [
+            "submitted_at_date_1",
+            "submitted_at_1" ,
+            "submitter_1",
+            "created_at_1",
+            "block_hash_1",
+            "remote_addr_1",
+            "peer_id_1",
+            "snark_work_1",
+            "graphql_control_port_1",
+            "built_with_commit_sha_1",
+            "state_hash_1", 
+            "parent_1",
+            "height_1",
+            "slot_1",
+            "validation_error_1"],
+        [
+            "submitted_at_date_2",
+            "submitted_at_2" ,
+            "submitter_2",
+            "created_at_2",
+            "block_hash_2",
+            "remote_addr_2",
+            "peer_id_2",
+            "snark_work_2",
+            "graphql_control_port_2",
+            "built_with_commit_sha_2",
+            "state_hash_2", 
+            "parent_2",
+            "height_2",
+            "slot_2",
+            "validation_error_2"]
+    ]
```

Comment:
I'm not sure about this. AWSKeyspacesClient().get_submissions() returns `List[Submission]` and this is an array of arrays. I think it should be something like:
```
submissions = [
   Submission(...),
   Submission(...),  
]